tee-worker/ts-tests/integration-tests/common/utils/context.ts

```diff
@@ -63,6 +61,22 @@ export async function initIntegrationTestContext(
     };
 }
 
+async function getWorkerEndpoint(api: ApiPromise): Promise<string> {
+    const registry = await api.query.teebag.enclaveRegistry.entries();
+    const identityEnclaves = registry.reduce((enclaves, [, enclave]) => {
+        if (enclave.isEmpty || !enclave.unwrap().workerType.isIdentity) {
+            return enclaves;
+        }
+        return [...enclaves, enclave.unwrap()];
+    }, [] as PalletTeebagEnclave[]);
+
+    if (identityEnclaves.length === 0) {
+        throw new Error('No identity worker found');
+    }
+
+    return identityEnclaves[Math.floor(Math.random() * identityEnclaves.length)].url.toHuman() as string;
```

Comment:
maybe the `.toString()` saves from forcing the type

Reply:
it needs to be casted as well because doing `toString()` to `AnyJson` returns `string | undefined`